crates/ruff_python_formatter/src/expression/expr_slice.rs

```diff
@@ -163,17 +163,23 @@ pub(crate) fn find_colons(
         .as_ref()
         .map_or(range.start(), |lower| lower.range().end());
     let first_colon =
-        first_non_trivia_token(after_lower, contents).ok_or(FormatError::SyntaxError)?;
+        first_non_trivia_token(after_lower, contents).ok_or(FormatError::SyntaxError {
+            message: "Din't find any token for slice first colon",
+        })?;
     if first_colon.kind != TokenKind::Colon {
-        return Err(FormatError::SyntaxError);
+        return Err(FormatError::SyntaxError {
+            message: "slice first colon token was not a colon",
+        });
```

Comment:
Nit: I like factory methods to make the construction slightly more ergonomic `FormatError::syntaxError(message)`

Reply:
done